web/src/main/java/com/microsoft/azure/simpletodo/model/TodoItem.java

```diff
@@ -4,28 +4,34 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.time.OffsetDateTime;
 
-import org.springframework.data.annotation.Id;
+import org.hibernate.annotations.GenericGenerator;
 import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
 import javax.validation.Valid;
 import javax.validation.constraints.*;
 
 
-import java.util.*;
 import javax.annotation.Generated;
 
 /**
  * A task that needs to be completed
  */
 
 @Generated(value = "org.openapitools.codegen.languages.SpringCodegen")
```

Comment:
FYI: Since the open api has been removed from this project, I'll delete all content about open api in another PR.
Here is screenthot about the search result about open api:
> ![image](https://user-images.githubusercontent.com/13167207/219534916-91d00143-65c2-4271-8651-7cf0ff19fb3d.png)